src/Compilers/CSharp/Test/Syntax/Parsing/ParserErrorMessageTests.cs

```diff
@@ -398,10 +398,7 @@ public static int Main ()
 }
 ";
 
-            CreateStandardCompilation(test).VerifyDiagnostics(
-                // (2,1): error CS0267: The 'partial' modifier can only appear immediately before 'class', 'struct', 'interface', or 'void'
-                // partial public class C  // CS0267
-                Diagnostic(ErrorCode.ERR_PartialMisplaced, "partial").WithLocation(2, 1));
+            CreateStandardCompilation(test).VerifyDiagnostics();
```

Comment:
In all of the cases where compilation now succeeds, I think we should access the type symbol for the declaration from the Compilation and verify that all the modifiers we expect are now present on the type.